spec/models/manageiq/providers/nuage/network_manager_spec.rb

```diff
@@ -72,7 +72,7 @@
       end.to raise_error(MiqException::MiqInvalidCredentialsError)
     end
 
-    context 'AMQP connection' do
+    context 'AMQP connection', :qpid_proton => true do
```

Comment:
If you throw a
```suggestion
    context 'AMQP connection', :qpid_proton => true do
      require 'qpid_proton'

```

here does that still pass for you without qpid_proton?  Since we're explicitly referencing `Qpid::Proton::Container` below I'm getting
```
Failures:

  1) ManageIQ::Providers::Nuage::NetworkManager validation AMQP connection handles connection errors
     Failure/Error: allow(Qpid::Proton::Container).to receive(:new).and_return(@conn)
     
     NameError:
       uninitialized constant Qpid
```

Reply:
The PR is green but I think it is sporadic, if I run this spec file directly and sometimes when running the whole suite it will fail with this error

Reply:
it doesn't like it out of the before block but works within the before.. pushing

Reply:
> The PR is green but I think it is sporadic, if I run this spec file directly and sometimes when running the whole suite it will fail with this error

yeah, good point... randomly getting qpid_proton required in other tests BEFORE this line would hide this error